oauthproxy.go

```diff
@@ -391,9 +394,12 @@ func buildPreAuthChain(opts *options.Options, sessionStore sessionsapi.SessionSt
 func buildSessionChain(opts *options.Options, provider providers.Provider, sessionStore sessionsapi.SessionStore, validator basic.Validator) alice.Chain {
 	chain := alice.New()
 
-	if opts.SkipJwtBearerTokens {
-		sessionLoaders := []middlewareapi.TokenToSessionFunc{
-			provider.CreateSessionFromToken,
+	if opts.SkipJwtBearerTokens || opts.SkipBearerTokens {
```

Comment:
We already considered renaming this flag in another PR. Please don't introduce a second SkipBearerToken. Just remove JWT from the current flag.

Reply:
On the other hand this will break current functionality. So maybe we need to keep the old one and mark it as deprecated.

Reply:
I put it in based on a comment in the PR, based on your suggestion I think :-)